.github/workflows/test.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                python-version: ['3.8', '3.9', '3.10']
```

Comment:
[That's a shame.](https://onnxruntime.ai/docs/get-started/with-python.html#supported-versions)

Reply:
Looks like the nightly builds support 3.10 though. If there's a way to check the python version in requirements.txt we could install that instead for 3.10

Reply:
Nah, let's wait until they release a stable version that supports 3.10. Supporting 3.10 isn't that important, especially since we push the integrated python anyway.

But could you please change the 3.8-3.10 support in the README?